aibolit/patterns/method_siblings/method_siblings.py

```diff
@@ -19,26 +19,30 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
-from typing import List
-from aibolit.types_decl import LineNumber
+from aibolit.ast_framework import ASTNodeType, AST
 from aibolit.utils.ast_builder import build_ast
-
-import javalang
+from aibolit.ast_framework.ast_node import ASTNode
+from typing import List
 import re
 
 
 class MethodSiblings:
-    def __init__(self):
-        pass
-
-    def value(self, filename: str) -> List[LineNumber]:
+    '''
+    Count those methods, which
+    start with the same names
+    '''
+    def __compare_next_nodes(self, node: ASTNode, new_node: ASTNode) -> List[int]:
```

Comment:
- Seems like we do not compare **next nodes**, but we compare **methods names**. You can mention this both in method name in method arguments.
 - The result of compression is usually `bool`, lines could be added to list in `value` method.

Reply:
Maybe even better name is `_is_method_names_close`.
Also I think for private methods _one_ underscore is enough.

Reply:
done